_posts/2020-03-06-strimzi-and-operator-lifecycle-manager.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title:  "Deploy Apache Kafka in few seconds with Strimzi and Operator Framework and Operator Lifecycle Manager"
+date: 2019-03-06
+author: jakub_scholz
+---
+
+[Operator Framework](https://github.com/operator-framework/) is an open source toolkit for writing and managing Kubernetes Operators.
+It consists of several components and one of them is the [Operator Lifecycle Manager (OLM)](https://github.com/operator-framework/operator-lifecycle-manager).
+OLM is an _Operator for Operators_.
+It provides a declarative way for installing and managing operators.
+It can make installation and management of Operators on Kubernetes much easier.
+And Strimzi is part of it.
+
+<!--more-->
+
+The main advantage for users from having Strimzi integrated with OLM is that they should be able to find Strimzi in different operator catalogs and install it directly from there.
+Users can also create their own catalogs with the operators they trust (and we hope Strimzi will be among them) and use OLM to manage them.
```

Comment:
I think it might be worth distinguishing end users from cluster administrators in this sentence.

```suggestion
Cluster administrators can also create their own catalogs with the operators they trust (and we hope Strimzi will be among them) and use OLM to let end users manage them.
```